src/umpire/TypedAllocator.inl

```diff
@@ -25,6 +25,13 @@ TypedAllocator<T>::TypedAllocator(Allocator allocator) :
 {
 }
 
+template<typename T>
+template<typename U>
+TypedAllocator<T>::TypedAllocator(const TypedAllocator<U>& other) :
+  m_allocator(other.m_allocator)
```

Comment:
Do we use this somewhere?

Reply:
It's used when creating a vector with a TypedAllocator with MSVC. Don't ask me why...